### DISCUSSION TITLE
white screen?

### DISCUSSION THREAD

**Discussion by Real-Yash** [stitionai/devika#432] [Q&A] [answered]
i have installed devika, python file is also running. but i am unable to access devika when i got to local host 3000 it is jst giving white screen and loading screen. and http://127.0.0.1:3000 is unable to reach can somebody explain what going on and how to fix it?

**Comment by alexanderbehrens1978** [comment, marked as answer]
I changed in devika.py
CORS(app,....) to CORS(app)
Now it works fine
Try llama2 and google search for the first. It is slow, but it works.

**Reply by Real-Yash** [reply]
It works now
